src/nemowallclock_meego.cpp

```diff
@@ -165,7 +165,11 @@ void WallClockPrivateMeego::settingsChanged(const Maemo::Timed::WallClock::Info
     info = newInfo;
 
     if (tzChange || tzaChange) {
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+        QQmlEngine *engine = qmlEngine(wallClock());
```

Comment:
I think this version should work on both qt5 and qt6. The qt5 header does "using namespace QtQml;"

I'd suggest just changing this single line first.